packages/ilib-lint/src/plugins/BuiltinPlugin.js

```diff
@@ -51,39 +51,39 @@ export const regexRules = [
         description: "Ensure that URLs that appear in the source string are also used in the translated string",
         note: "URL '{matchString}' from the source string does not appear in the target string",
         regexps: [ "((https?|github|ftps?|mailto|file|data|irc):\\/\\/)([\\da-zA-Z\\.-]+)\\.([a-zA-Z\\.]{2,6})([\\/\w\\.-]*)*\\/?" ],
-        link: "https://github.com/ilib-js/ilib-lint/blob/main/docs/resource-url-match.md"
+        link: "https://github.com/ilib-js/ilib-mono/ilib-lint/blob/main/docs/resource-url-match.md"
```

Comment:
Also, 404 file not found.

```suggestion
        link: "https://github.com/ilib-js/ilib-mono/blob/main/packages/ilib-lint/docs/resource-url-match.md"
```
This same change applies to all of the "blob/main" links in these files

Reply:
Fixed in 23a45e9